.mvn/maven.config

```diff
@@ -0,0 +1,3 @@
+-Pconsume-incrementals
+-Pmight-produce-incrementals
```

Comment:
This is to set up Incrementals. You have not done that: https://github.com/jenkinsci/webhook-step-plugin/blob/a707a7cbebe08f2dae1e08d7bf8ae716472f1511/pom.xml#L13